srcpkgs/grub/template

```diff
@@ -63,21 +64,25 @@ do_configure() {
 	sed -i 's|/usr/share/fonts/unifont|/usr/share/fonts/misc|' configure
 
 	# build tools
-	_configure_args=${configure_args}
-	if [ -n "${_NATIVE_PLATFORM}" ]; then
-		_configure_args="--with-platform=${_NATIVE_PLATFORM} ${_configure_args}"
-	fi
 	msg_normal "Configuring grub tools...\n"
 	mkdir $wrksrc/build
 	cd $wrksrc/build
-	../configure --host=${XBPS_TARGET_MACHINE} ${_configure_args}
+	../configure --host=${XBPS_TARGET_MACHINE} ${configure_args} \
+		${_NATIVE_PLATFORM:+--with-platform=${_NATIVE_PLATFORM}}
 
 	for _SUPP_BUILD in ${_SUPPLEMENTARY_BUILDS}; do
 		_TARGET=${_SUPP_BUILD%%-*}
 		_PLATFORM=${_SUPP_BUILD##*-}
 		msg_normal "Configuring ${_TARGET} ${_PLATFORM} grub...\n"
 		mkdir $wrksrc/${_PLATFORM}_${_TARGET}_build
 		cd $wrksrc/${_PLATFORM}_${_TARGET}_build
+		if [ "$_TARGET" = x86_64 ] &&
+		   [ "${XBPS_TARGET_MACHINE%-musl}" = i686 ]; then
+			_TARGET=x86_64-linux-musl
+			CFLAGS="${BASE_CFLAGS/-mtune=i686/-mtune=generic}"
```

Comment:
i don't like this weird block/substitution, considering the extra platform files are basically arch-independent, there should be no harm in simply configuring them all with empty `CFLAGS`

Reply:
that is not what you did there...

Reply:
Sorry, but I can't get what you meant!

Reply:
now it looks fine